internal/update/update.go

```diff
@@ -0,0 +1,186 @@
+//nolint:errcheck
+package update
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/open-policy-agent/opa/rego"
+
+	_ "embed"
+)
+
+//go:embed update.rego
+var updateModule string
+
+const CheckAndWarnIgnoreVariable = "REGAL_FEATURES_REMOTE_CHECK_VERSION"
```

Comment:
The const is called IgnoreVariable, but the env var name is a feature flag... So it's unclear if this is something to opt-out or to opt-in to, from the env name alone. 

My recent favorite telemetry transparency example is observable framework: https://observablehq.com/framework/telemetry. They have a `_DISABLE` var:
> You can [opt-out of telemetry](https://observablehq.com/framework/telemetry#disabling-telemetry) by setting the `OBSERVABLE_TELEMETRY_DISABLE` environment variable to `true`.

Reply:
Thanks, I have adjusted the name for this one now.